### ISSUE TITLE
Display bigger textarea for editing static pages

### ISSUE THREAD

**Issue by unho** [translate/pootle#4746]
The textarea to edit a staticpage is sometimes too small. I recommend that the textarea with the source to use half the height the browser viewport has, with a minimum height of `15 em`.

![edit_staticpage](https://cloud.githubusercontent.com/assets/939880/15453891/df5004d4-2026-11e6-9d0e-960736cf834a.png)

In my current setup it is using `100px` which barely allows to see four lines:

![current](https://cloud.githubusercontent.com/assets/939880/15453914/df802e1a-2027-11e6-88d9-84cb0a056d5c.png)

**Comment by iafan** [comment]
I think the better way of handling this at smaller window sizes would be having two tabs: `Write | Preview` (like on GitHub) and having their height adjusted to approx 90% of window height. On wide screens, as it is now, these two tabs will be separated into two independent panes.

**Comment by unho** [comment]
Displaying both `Write | Preview` as they are now, but with larger heights (`15em` which are roughtly `300px` in my setup) does allow to see them both completely at the same time. Note that when editing the text in the staticpage/announcement the rest of the edit form (title, is_active) is actually not needed. I don't mind having to scroll down to fully see the `Write | Preview`, but what is awfull is having to scroll within the `Write` to see what you are typing at all.

Using a GitHub like approach defeats the originally idea of allowing get an immediate visual feedback of what you are typing. But for window sizes smaller than mine might be a valid approach.

**Comment by iafan** [comment]
Yes, that's for smaller screens. I'd say that when the editbox contains less than 10 lines, and the page is not wide enough to show two panes side-by-side horizontally, it's better to show tabs.
